### PR TITLE
Fix 0009-append-flatten expressions compatibility decl. type tNestedList

### PR DESCRIPTION
**Pull request by tarilabs**
Currently the following node/decision:
`append2`
`append3`
`append4`

do evaluate to `[a, b, c, [[w, x], y, z]]`

This is not compatible with declared type `tNestedList`, being the
actual value above a list-of-tNestedList.

In order to make the test compatible with the declared type, it is 
proposed to change the expression in order to make them evaluate to
`[[w, x], y, z, [a, b, c]]`
which is compatible with the current declared type `tNestedList`.

In other words, instead of appending a `tNestedList` inside a
`tStringList`, and inadvertently create a 3 level deep list, change
affected expressions to appending a `tStringList` inside a `tNestedList`.
This is because `tNestedList` elements are indeed expected to be of type
`tStringList`.

### CODE REVIEW

**Comment by etirelli** [comment]
@tarilabs the content of the changes look good, but there is a problem with the formatting of the code (indentation). Can you take a look if you can fix it please?

**Comment by tarilabs** [comment]
I've realigned with the original indentation containing a mix of TABs and spaces, so now the diff should show only semantic changes.
